src/ontology/cl-edit.owl

```diff
@@ -31843,6 +31847,14 @@ AnnotationAssertion(rdfs:label obo:CL_4033053 "small bistratified retinal gangli
 SubClassOf(obo:CL_4033053 obo:CL_0003001)
 SubClassOf(obo:CL_4033053 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001792))
 
+# Class: obo:CL_4033054 (perivascular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24928499") Annotation(oboInOwl:hasDbXref "PMID:27889329") obo:IAO_0000115 obo:CL_4033054 "A cell that is adjacent to a vessel. A perivascular cell plays a crucial roles in maintaining vascular function and tissue homeostasis. This cell regulates vessel integrity and flow dynamics.")
```

Comment:
A perivascular cell plays a crucial roles in -->
A perivascular cell plays a crucial **role** in
or
A perivascular cell **plays crucial** roles in